SettingsPages&ForgotPassword

ag3722 opened the pull request:
- Merchant Registration
- Profile Page
- Business Info Settings
- Personal Info Settings
- Security Settings
- Forgot Password

ag3722 committed:
SettingsPages&ForgotPassword

- Merchant Registration
- Profile Page
- Business Info Settings
- Personal Info Settings
- Security Settings
- Forgot Password

SaifA21 commented on app/forgot_password.tsx:
```diff
@@ -27,69 +30,57 @@ export default function forgotPassword() {
-
+      <View style={styles.header}>
+        <TouchableOpacity onPress={() => navigation.goBack()} style={styles.pageBackButton}>
+          <Icon name="arrow-back-outline" size={24} color="#000" />
```
the back button does not take you back to the login page

SaifA21 commented on app/(auth)/security.tsx:
for security  (change password) we actually can't let them change the password in the app, essentially this should send them an email just like it does on the login page and then they could change it there

SaifA21 commented on app/(auth)/(tabs)/profile.tsx:
```diff
@@ -75,15 +79,14 @@ export default function Profile() {
+        <SettingItem title="Business Info" icon="store" path="BusinessSettings" />
+        <SettingItem title="Account Info" icon="person" path="PersonalSettings" />
+        <SettingItem title="Security" icon="lock" path="security" />
+        <SettingItem title="Language" icon="language" path="home" />
```
lets get rid of language since we won't support any other languages

SaifA21 reviewed (changes requested):
Left some comments for changes

ag3722 committed:
PR Changes

- Removed language from profile page
- Edit password changed to email link
- Verified back button functionality on forgot password page

ag3722 commented:
I've made the requested changes

SaifA21 reviewed (approved):
LGTM
